src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,26 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        Set<Character> setOfChars = new TreeSet<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            int value = reader.read();
+            while (value != -1) {
+                if (Character.isLetter((char) value)) {
+                    setOfChars.add((char) Character.toLowerCase(value));
+                }
+                value = reader.read();
+            }
+        } catch (IOException e) {
+            throw new FileNotFoundException("The file cannot be read");
```

Comment:
all of the possible IOExcetion will be wrapped into FileNotFoundException. This will confused somebody who is gonna read some logs. In this case you may just leave throws near method signature

Reply:
>In this case you may just leave throws near method signature

Do you mean I should remove the catch block altogether and say that in method signature `throws IOException`?